### PR TITLE
LineEdit: password mode, linking. PasswordDialog: use LineEdit

### PR DESCRIPTION
**Pull request by xiphon**
Related: https://github.com/monero-project/monero-gui/issues/2700
monero-gui-password

### CODE REVIEW

File: components/LineEdit.qml [hunk 1]
```diff
@@ -210,6 +239,27 @@ Item {
             onTextChanged: item.textUpdated()
             topPadding: 10
             bottomPadding: 10
+            echoMode: isPasswordHidden() ? TextInput.Password : TextInput.Normal
+
+            MoneroComponents.Label {
+                visible: password || passwordLinked
+                fontSize: eyeMouseArea.containsMouse ? 24 : 20
```
Comment (selsta) [inline components/LineEdit.qml:246]:
```suggestion
                fontSize: eyeMouseArea.containsMouse ? 22 : 20
```

24 feels a bit too much

Comment (xiphon) [reply components/LineEdit.qml:246]: Is a copy-paste of the current code.

TBH, I don't get why it gets bigger on hover. Would prefer to drop this. Thoughts?

Comment (selsta) [reply components/LineEdit.qml:246]: Yes, I prefer dropping this and only changing color.

Comment (xiphon) [reply components/LineEdit.qml:246]: Updated

**Pull request merged**